### PR TITLE
feat: [CVS-9008]: Integrate custom metric in NewRelic

### PR DESCRIPTION
**Pull request by deepesh-ui**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by deepesh-ui** [comment]
retrigger coverage

**Comment by deepesh-ui** [comment]
retrigger jest

**Comment by deepesh-ui** [comment]
trigger cypress

**Comment by deepesh-ui** [comment]
retrigger jest

File: cypress/integration/85-cv/HealthSource/NewRelic.spec.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+    cy.contains('div', 'NewRelic HS').click({ force: true })
+    cy.contains('span', 'Next').click()
+
+    cy.get('input[name="newRelicApplication"]').should('have.value', 'My Application')
+    cy.get('input[name="metricData.Performance"]').should('be.checked')
+    cy.contains('span', 'Submit').click({ force: true })
+  })
+
+  it('Add new AppDynamics monitored service with custom metric', () => {
```
Comment (harshilbits) [inline cypress/integration/85-cv/HealthSource/NewRelic.spec.ts:78]: Please update AppD with New Relic everywhere.

File: src/modules/85-cv/pages/health-source/connectors/NewRelic/NewRelicHealthSource.tsx [hunk 2]
```diff
@@ -270,17 +287,40 @@ export default function NewRelicHealthSource({
               </Layout.Vertical>
             </CardWithOuterTitle>
             {showCustomMetric ? (
-              <NewRelicMappedMetric
+              // <NewRelicMappedMetric
```
Comment (harshilbits) [inline src/modules/85-cv/pages/health-source/connectors/NewRelic/NewRelicHealthSource.tsx:290]: pls remove commented code

File: cypress/integration/85-cv/HealthSource/NewRelic.spec.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    cy.intercept('GET', applicationCall, applicationResponse).as('ApplicationCall')
+    cy.intercept('GET', metricPackCall, metricPackResponse).as('MetricPackCall')
+    cy.intercept('GET', metricDataCall, metricDataResponse).as('MetricDataCall')
+
+    cy.addNewMonitoredServiceWithServiceAndEnv()
+
+    cy.contains('span', 'Add New Health Source').click()
+
+    // Fill Define HealthSource Tab with NewRelic
+    cy.get('span[data-icon="service-newrelic"]').click()
```
Comment (harshilbits) [inline cypress/integration/85-cv/HealthSource/NewRelic.spec.ts:41]: Fill Define HealthSource Tab with NewRelic can be made generic function the way we have cy.visitChangeIntelligence().

This function can be reused all connectors.
This function will take inputs icon, healthsourcename and feature name

**Review by harshilbits — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**